src/main/java/tracing/SNT.java

```diff
@@ -0,0 +1,26 @@
+package tracing;
+
+import org.scijava.util.VersionUtils;
+
+import ij.IJ;
+
+/** Static utilities for SNT **/
+public class SNT {
+	public static final String VERSION = getVersion();
+
+	private SNT() {
+	}
+
+	private static String getVersion() {
+		return VersionUtils.getVersion(tracing.SimpleNeuriteTracer.class);
+	}
+
+	protected static void error(final String string) {
+		IJ.error("Simple Neurite Tracer v" + getVersion(), string);
+	}
+
+	protected static void log(final String string) {
+		System.out.println("[SNT] " + string);
```

Comment:
@imagejan, @ctrueden: Yeah, that was the idea behind this code cleanup: Move all the hundreds of ```System.out.println``` calls in Simple Neurite Tracer to a single utility class so that things can be more more customizable. I started using [LogService in BAR](https://github.com/tferr/Scripts/blob/master/BAR/src/main/java/bar/Runner.java#L58-L80) while ago, and thought about it here but the utility [SNT class](https://github.com/tferr/Simple_Neurite_Tracer/blob/dev/src/main/java/tracing/SNT.java)  just contains static methods and Simple Neurite Tracer is a mammoth of a plugin stuck in IJ1. The only way I thought about implementing it (after e.g., this [forum thread](http://forum.imagej.net/t/gui-stalling-and-eventually-freezing-while-trakem2-generates-mipmaps/?u=tferr)), was using something like:

```java
@Plugin(type = ContextCommand.class)
public class SNT {

	@Parameter
	private static Context context;

	@Parameter
	private static LogService logService;

	private SNT() {
		// private constructor to prevent instantiation of class
	}
	
	private static void initLog() {
		if (context == null)
			context = (Context) IJ.runPlugIn("org.scijava.Context", "");
		if (logService == null)
			logService = context.getService(LogService.class);
	}

	protected static void log(final String string) {
		if (logService == null)
			initLog();
		logService.debug("[SNT] " + string);
	}

}
```
Would you guys be OK with something like the above? Is this even good practice?